tests/daemon/test-persistent-cache.c

```diff
@@ -987,7 +976,7 @@ test_persistent_cache_does_not_compute_offset_when_boot_id_is_same (gboolean
 
   // This call should have to compute the boot offset itself.
   get_succeeded =
-    emer_persistent_cache_get_boot_time_offset (cache2, NULL, TRUE, &error);
+    emer_persistent_cache_get_boot_time_offset (cache2, NULL, &error);
```

Comment:
This call can be removed, and the comment above can be moved to describe `EmerPersistentCache *cache2 = make_testing_cache ();` (pre-existing defect).